### DISCUSSION TITLE
Detailsuche liefert zusätzliches falsches Ergebnis

### DISCUSSION THREAD

**Discussion by stpgis** [e-netze/webgis-community#177] [General]
Wenn wir bei der Detailsuche Straßenname die Waldstraße aus dem Listenfeld auswählen kommt bei der Selektion auch die Thurnwaldstraße mit siehe Screenshot. 
![grafik](https://github.com/gis-eni/webgis-forum/assets/171917921/e63d1ae9-e4d4-4f8e-93b9-27f376ab8a7d)

**Comment by gis-eni** [comment]
Da kann ich wenig dazu sagen. Ich weiß nicht was du da genau parametriert hast, welche Art von Dienst dahinter steht, etc.
Die diese Einstellungen zu vertraulich sind, kannst mir die Informationen auch per schicken.

**Comment by gis-eni** [comment]
Lösung: Wildcard vorne weg ;)
